lxd/instance/drivers/driver_qemu.go

```diff
@@ -529,6 +529,32 @@ func (d *qemu) Freeze() error {
 	return nil
 }
 
+// configDriveMountPath returns the path for the config drive bind mount.
+func (d *qemu) configDriveMountPath() string {
+	return filepath.Join(d.DevicesPath(), "config")
```

Comment:
What happens if I happen to define a device called `config` that's also a disk but a different one?

Reply:
I suspect the safe thing to do here is to mount it as a sub-directory of the instance mountpoint instead (same location where the writable config path is). So something like `config.mount` alongside the writable `config` directory.